ci/jobs-dev/run_ifi_standalone_rrfs_URSA.sh

```diff
@@ -36,15 +36,15 @@ module list
 ulimit -s unlimited
 ulimit
 
-msg="Starting ifi_standalone_fv3r test"
+msg="Starting rrfs_ifi test"
```

Comment:
```suggestion
msg="Starting ifi_standalone_rrfs test"
```
@SamuelTrahanNOAA To ensure consistency between run_ifi_standalone_hrrr_URSA.sh and run_ifi_standalone_hrrr_URSA.sh